atom/browser/native_window_mac.mm

```diff
@@ -382,6 +393,20 @@ - (void)drawRect:(NSRect)dirtyRect {
   // We will manage window's lifetime ourselves.
   [window_ setReleasedWhenClosed:NO];
 
+  // Configure title bar look on Yosemite or newer
+  if (base::mac::IsOSYosemiteOrLater()) {
+    if ((titleBarStyle == "hidden") || (titleBarStyle == "hidden-inset")) {
+      [window_ setTitlebarAppearsTransparent:YES];
```

Comment:
This throws on OS X Mavericks with Electron 0.30.3 when `titleBarStyle == "hidden-inset"`. I thought the `base::mac::IsOSYosemiteOrLater()` would guard against that?

![image](https://cloud.githubusercontent.com/assets/170270/9879233/a92dd8e0-5bf0-11e5-870a-a3fbc3656243.png)